src/main/webapp/app/core/auth/account.service.ts

```diff
@@ -333,4 +333,11 @@ export class AccountService implements IAccountService {
         }
         return this.http.put<void>('api/users/sshpublickey', sshPublicKey);
     }
+
+    deleteSshPublicKey() {
+        if (this.userIdentity) {
+            this.userIdentity.sshPublicKey = '';
```

Comment:
Why an empty string?
```suggestion
            this.userIdentity.sshPublicKey = undefined;
```

Reply:
personal taste I guess